src/datalayer/actions/event.action.js

```diff
@@ -0,0 +1,12 @@
+import { eventAction, authAction } from 'constants/actions';
+import { get } from 'utils/request';
+import localConfig from 'configs/local';
+
+export const changeToLoginComponent = () => ({
+  type: authAction.IS_LOGIN_COMPONENT,
+});
+
+export const loadEventsListDispatch = (uid) => ({
+  type: eventAction.LOAD_EVENTS_LIST,
+  promise: get(`${localConfig.apiUrl}/users/${uid}/events`, {}, true),
```

Comment:
Check logic at request.js. We have to put apiUrl in utils. And import from configs, not localConfig

Reply:
Oke mb